docs/staticAnalysis.md

```diff
@@ -103,6 +119,20 @@ To set it up, [install node.js](https://nodejs.org/en/download/) if necessary (v
 npm install -g eslint@{version}
 ```
 
+#### Installing ESLint from within IntelliJ
+1. Ensure the [NodeJS Plugin](https://plugins.jetbrains.com/idea/plugin/6098-nodejs) is installed.
+1. Refer to [this guide](https://www.jetbrains.com/help/idea/2016.3/using-javascript-code-quality-tools.html#installESLint) to install ESLint.
```

Comment:
Consider linking to https://www.jetbrains.com/help/idea/2016.3/using-javascript-code-quality-tools.html#ESLint instead so that readers dont miss instructions of installing nodejs plugin.